src/lib/utils/meta_tables/meta_temporary_memory_usage_table.cpp

```diff
@@ -0,0 +1,37 @@
+#include "meta_temporary_memory_usage_table.hpp"
+
+#include <chrono>
+#include <magic_enum.hpp>
+
+#include "hyrise.hpp"
+
+namespace opossum {
+
+MetaTemporaryMemoryUsageTable::MetaTemporaryMemoryUsageTable()
+    : AbstractMetaTable(TableColumnDefinitions{{"operator_type", DataType::String, false},
+                                               {"operator_data_structure", DataType::String, false},
+                                               {"timestamp", DataType::Long, false},
+                                               {"amount", DataType::Long, false}}) {}
```

Comment:
Amount is usually used for non-countable things. Why not something explicit like `memory_consumption_in_bytes`?